### ISSUE TITLE
Final Check on descriptions for RIAW

### ISSUE THREAD

**Issue by terracoda** [phetsims/resistance-in-a-wire#122]
To make sure the descriptions meet the pedagogical goals for this sim, I need a
1. A sanity check from content experts, and 
2. I would like to request Design Time for Next Week (not this week)

The design doc with the descriptions can be found here:
[https://docs.google.com/spreadsheets/d/11OoUnYOpeEYojNxCOMyyOyEQfO4tdkFWEGJbt-qwqsQ/edit#gid=0](https://docs.google.com/spreadsheets/d/11OoUnYOpeEYojNxCOMyyOyEQfO4tdkFWEGJbt-qwqsQ/edit#gid=0)

I seem to have an idea that there is something missing in the alerts, maybe something about a 3rd letter, or something about the wire?

**Comment by terracoda** [comment]
@ariel-phet, when you have a bit of time we can look at the spread sheet together.
@emily-phet please add anything if I have forgotten to mention anything this issue.

**Comment by ariel-phet** [comment]
@terracoda I would be happy to. How pressing is this?

**Comment by terracoda** [comment]
Somewhat pressing. Are you available Friday morning? 

@emily-phet wants to identify any description design issues that I can quickly address, so @jessegreenberg implement and we can get the sim re-published with description.

**Comment by emily-phet** [comment]
@ariel-phet @terracoda @jessegreenberg 
I think there has been some miscommunication. I don't think this discussion is urgent. I think it can all be discussed in a design meeting next week. 

@terracoda The things that you mention in your list were intended to be done at the design meeting next week - not to have done in advance of a design meeting.

**Comment by terracoda** [comment]
Oh sorry, my apologies. We will discuss at the design meeting next week @ariel-phet .

.
.

**Comment by emily-phet** [comment]
@terracoda @ariel-phet during design meeting with this sim, let's spend a little time exploring with NVDA (PC folks) or VO (Mac folks). This is a great opportunity for folks to start learning the basics, and will help with productive feedback for @terracoda

**Comment by terracoda** [comment]
@emily-phet that's a great idea, but I don't yet have a version with any implemented descriptions. 
I thought we were going to go over the descriptions for any design gaps and then implement?

I'm just checking with @jessegreenberg in case I am missing a version somewhere.

**Comment by emily-phet** [comment]
@terracoda we can use Ohm's Law as an example, since they're so similar. I think that we should start - as a general practice - listening to screen readers do their thing at least a little when doing these description design meetings with the broader team. I think that will take some of the verbal load off of you (I know it's really hard for me at least to have to phrase things as screen readers do!), and it will be good practice for all the team members involved.

**Comment by terracoda** [comment]
Awesome, I was thinking the same thing. Demo, OL, and then look at the RIAW design spreadsheet and identify the gaps or issues there before full implementation.

**Comment by emily-phet** [comment]
discussed in 1/25 design meeting. notes in @terracoda's RIAW description spreadsheet.

**Comment by terracoda** [comment]
@terracoda is updating the design document and may still have a couple of outstanding questions.

**Comment by terracoda** [comment]
@terracoda has updated the google sheet containing all the descriptions and alerts for RIAW. These descriptions can be implemented with one global change - descriptions and alerts should say "ohms" everywhere except when Resistance actually has the value of 1 ohm. if you can't do the exception, "ohms" is to be used everywhere.

Please note that there will be a couple of visual design changes required as well:
1. In the visual sim, for the value of Resistance, the word "ohm" needs to be changed to "ohms".
2. The visual of Resistance should never reach 0.00. This is a rounding error. 

I will open two issues for these two items.

@emily-phet, I am reassigning to @jessegreenberg for implementation.

**Comment by jessegreenberg** [comment]
Thanks @terracoda. I am going to create a new issue for implementation since this one was about checking the design. Closing.
